compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/impl/SymbolFactory.java

```diff
@@ -612,6 +618,33 @@ public BallerinaXMLNSSymbol createXMLNamespaceSymbol(BXMLNSSymbol symbol) {
         return symbolBuilder.build();
     }
 
+    /**
+     * Creates a client declaration Symbol.
+     *
+     * @param symbol declaration symbol to convert
+     * @return {@link BallerinaClientDeclSymbol}
+     */
+    private BallerinaClientDeclSymbol createClientDeclSymbol(BClientDeclarationSymbol symbol) {
+        BallerinaClientDeclSymbol.ClientDeclSymbolBuilder symbolBuilder =
+                new BallerinaClientDeclSymbol.ClientDeclSymbolBuilder(symbol.getName().getValue(), symbol, context);
+
+        for (AnnotationAttachmentSymbol annot : symbol.getAnnotations()) {
+            symbolBuilder.withAnnotation(createAnnotationSymbol((BAnnotationAttachmentSymbol) annot));
+        }
+        return symbolBuilder.build();
```

Comment:
```suggestion
        }
        
        return symbolBuilder.build();
```

Better add a new line.